Assets/Scripts/Room.cs

```diff
@@ -19,9 +19,30 @@ public class Room : MonoBehaviour
 
     public UnityEvent<string> roomEnteredEvent = new UnityEvent<string>();
 
+    public string roomtype = "normal";
+
+    private int enemy_count;
+
+    private GameObject[] enemylist;
+
     // Start is called before the first frame update
     void Start()
-    {
+    {   
+        if (roomtype.Equals("normal")){
+            enemy_count = (int) Random.Range(2,6);
+            enemylist = Resources.LoadAll<GameObject>("Prefabs/Enemy");
+
+            for (int i=0; i<enemy_count; i++){
+                Debug.Log("Creating enemy number: " + i);
+                int index = Random.Range(0,enemylist.Length);
+                GameObject enemy = Instantiate(enemylist[index]);
+                enemy.SetActive(false);
+                enemy.transform.position = new Vector3(transform.position.x + Random.Range(-10f,10f),transform.position.y + Random.Range(-4f,4f),12.07283f);
```

Comment:
suggest querying the bounds of this Room's gameobject to scale the Random.Range offsets here instead of relying on 10 and 4, and replacing literal numbers with named constants

Reply:
I tried two script ways and none is working. Not going to spend too much time on this.

Reply:
Okay be sure you own this code if we get to random map generation, those offsets could place enemies out of bounds if you generate a small enough room